docs/graphify-local-embedding-pass.md

```
@@ -0,0 +1,489 @@
+    d = base / "cache" / kind
+```
+
+So the embedding cache is `kind="embeddings"` →
+`graphify-out/cache/embeddings/`, sitting beside `ast/`, `semantic/`, and
+`semantic-deep/`.
+
+**Key** — deliberately *not* graphify's `file_hash` (which is
+`SHA256(file_bytes || \0 || relpath)` and keyed to a *file*). An embedding is
+keyed to a *node's embedded text*:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)